Source/Dafny/DafnyOptions.cs

```diff
@@ -309,6 +310,15 @@ public enum IncludesModes {
             return true;
           }
 
+        case "mainArgs":
+        case "MainArgs": {
```

Comment:
Is there a reason to include this uppercase version? I think the convention for Dafny option is lowerCamelCase.

Reply:
I saw in some places support for lowerCamelCase and UpperCamelCase, so I'd think I'd mimick it. No strong reason to do it that way though.